blocks/header/header.css

```diff
@@ -131,145 +230,196 @@ header nav[aria-expanded='true'] .nav-hamburger-icon::after {
   transform: rotate(-45deg);
 }
 
-@media (width >= 900px) {
-  header nav .nav-hamburger {
-    display: none;
-    visibility: hidden;
-  }
-}
-
-/* brand */
-header .nav-brand {
-  grid-area: brand;
-  flex-basis: 128px;
-  font-size: var(--heading-font-size-s);
-  font-weight: 700;
-  line-height: 1;
+/* nav-drop */
+header .nav-drop {
+  position: relative;
+  cursor: pointer;
 }
 
-header nav .nav-brand img {
-  width: 128px;
-  height: auto;
+header .nav-drop::after {
```

Comment:
The icon is not aligned properly on mobile
<img width="169" alt="obraz" src="https://github.com/user-attachments/assets/a200c218-4b6e-4322-9bc7-5480f22aea7e">

Reply:
And it is overlapping for 375px devices